Make website more responsive, remove Christmas trees from lyrics.

TechStudent11 opened the pull request:
Fixes #80
Fixes #60

KendallDoesCoding commented:
@TechStudent11 just make it so that if the screen is smaller then the normal computer website size, it doesnt show the trees and then it will be good to merge! The rest seems to be good only trees in the way.

KendallDoesCoding commented:
Yo, you working on this @TechStudent11

KendallDoesCoding commented:
@TechStudent11 When will you be done with this?

TechStudent11 commented:
Sorry! Currently doing classes.

TechStudent11 commented:
@KendallDoesCoding Works now.

KendallDoesCoding commented:
@TechStudent11 Thanks for removing the trees in lyrics, but in the main directory I need you to make it so that it will remove the trees but only if the screen size is small because still now the trees are covering the screen for small phones.

KendallDoesCoding commented:
@TechStudent11 Also what I meant by #60 is to remove the trees from this page: https://deploy-preview-83--mogulchristmas.netlify.app/lyrics not the trees from all the songs, would appreciate you to change it back after doing the above changes I requested, but if you cannot now that's all good too, there's no harm if there are no trees in the songs

TechStudent11 commented:
@KendallDoesCoding 

> @TechStudent11 Thanks for removing the trees in lyrics, but in the main directory I need you to make it so that it will remove the trees but only if the screen size is small because still now the trees are covering the screen for small phones.

I broke that by accident. Now fixed.

> @TechStudent11 Also what I meant by #60 is to remove the trees from this page: https://deploy-preview-83--mogulchristmas.netlify.app/lyrics not the trees from all the songs, would appreciate you to change it back after doing the above changes I requested, but if you cannot now that's all good too, there's no harm if there are no trees in the songs

Fixed as well.

KendallDoesCoding commented:
> 

Will check it out in a bit. Thanks

KendallDoesCoding reviewed (changes requested):
For some reason, in the mobile mode in the inspect element the songs arent playing, but that seems to be a inspect element glitch, but when the tab is small its playing. I also want to dark mode button moved somewhere if the screen size is small coz its merging with "A Very Mogul Christmas" or maybe just make the button smaller, if not it can be moved below "A Very Mogul Christmas" before the line "Christmas Album by Ludwig".

TechStudent11 commented:
> For some reason, in the mobile mode in the inspect element the songs arent playing

?

> but that seems to be a inspect element glitch

what glitch?

> but when the tab is small its playing

?

> I also want to dark mode button moved somewhere if the screen size is small coz its merging with "A Very Mogul Christmas" or maybe just make the button smaller, if not it can be moved below "A Very Mogul Christmas" before the line "Christmas Album by Ludwig".

That is the only thing I understand.

KendallDoesCoding commented:
Also, you linked #60 but that issue is mainly to clear up the css and make it only into one file. I was thinking of closing the issue and making another issue for the trees being in the lyrics directory, so I'm unlinking that issue for now and will link another issue shortly.

TechStudent11 commented:
> > > For some reason, in the mobile mode in the inspect element the songs arent playing
> > 
> > 
> > ?
> > > but that seems to be a inspect element glitch
> > 
> > 
> > what glitch?
> > > but when the tab is small its playing
> > 
> > 
> > ?
> > Nothing to worry, in the inspect element I went into mobile mode and the songs weren't playing but while inspect element was open I made the page small and it was responsive only issue I had was
> > > I also want to dark mode button moved somewhere if the screen size is small coz its merging with "A Very Mogul Christmas" or maybe just make the button smaller, if not it can be moved below "A Very Mogul Christmas" before the line "Christmas Album by Ludwig".

What

KendallDoesCoding commented:
> > > > For some reason, in the mobile mode in the inspect element the songs arent playing
> > > 
> > > 
> > > ?
> > > > but that seems to be a inspect element glitch
> > > 
> > > 
> > > what glitch?
> > > > but when the tab is small its playing
> > > 
> > > 
> > > ?
> > > Nothing to worry, in the inspect element I went into mobile mode and the songs weren't playing but while inspect element was open I made the page small and it was responsive only issue I had was
> > > > I also want to dark mode button moved somewhere if the screen size is small coz its merging with "A Very Mogul Christmas" or maybe just make the button smaller, if not it can be moved below "A Very Mogul Christmas" before the line "Christmas Album by Ludwig".
> 
> What

Forget it lol, just do "I also want to dark mode button moved somewhere if the screen size is small coz its merging with "A Very Mogul Christmas" or maybe just make the button smaller, if not it can be moved below "A Very Mogul Christmas" before the line "Christmas Album by Ludwig"."

KendallDoesCoding commented:
Lol, don't worry about merging the branch I just did some changes to README.md nothing important.

KendallDoesCoding commented:
@TechStudent11 I don't think it is complete but incase you didnt realize rn, its opening the tab in darkmode for anyone its not if they've set it the previous time they came to the site.

aliraza944 commented:
No, I think the @TechStudent11 should do it. It's just adjusting the size of the button on different screens. Start with the mobile one and then work your way up. I am working on the #38

KendallDoesCoding commented:
> No, I think the @TechStudent11 should do it. It's just adjusting the size of the button on different screens. Start with the mobile one and then work your way up. I am working on the #38

Yeah, he did remove the christmas tree for small sizes in his PR #83 so he can do this too, but can use your help if needed. Hope your comfortable doing this @TechStudent11.

KendallDoesCoding commented:
@TechStudent11 Can you please work on the responsive changes I requested and #98?

KendallDoesCoding commented:
@TechStudent11 Reminder on this, please complete this and mark it as "Ready for review" as soon as possible.

TechStudent11 commented:
k

KendallDoesCoding commented:
@TechStudent11 If you don't wanna work on #98 for now its ok, please just remove your changes made for that and update you description of the PR removing that line, cause I think it does fix #80 and #60 and I wanna add the features of those issues urgently in the next release, #98 can wait for a couple releases that's not too important, but especially #60 I need to add urgently.

TechStudent11 commented:
Okay I'll revert it

KendallDoesCoding commented:
> Okay I'll revert it

Thanks! Just revert it using Github Desktop and I'll merge this and later you or @aliraza944 can make another PR to work on #98!

TechStudent11 commented:
All good.

KendallDoesCoding commented:
> Owner

Actually, forgot I asked for this to be done @TechStudent11 , can you please work on making the toggle button small, I'll merge this and then you can work on #98 in another PR when complete.

KendallDoesCoding commented:
> > > Owner
> > 
> > 
> > Actually, forgot I asked for this to be done @TechStudent11 , can you please work on making the toggle button small, I'll merge this and then you can work on #98 in another PR when complete.
> 
> @TechStudent11 Let me know, if you can work on this now, otherwise I'll merge this for now and you can work on this later.... Thanks

Ok, I'm just merging this for now, you can work on the making the toggle button size small in a bit @TechStudent11 or @aliraza944
